File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
using namespace std;

class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int n1 = word1.size();
        int n2 = word2.size();
        string merge = "";
        
        int i = 0;
        while(i < n1 || i < n2){
            if (i < n1){
               merge += word1[i]; 
            }
                
            if (i < n2){
                merge += word2[i];
            }
                
            i += 1;
        }
            
            
                
        return merge;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n1 = len(word1)
        n2 = len(word2)
        merge = ""
        
        i = 0
        while i < n1 or i < n2:
            if i < n1:
                merge += word1[i]
            if i < n2:
                merge += word2[i]
            i += 1
            
                
        return merge
        